source/grid/reference_cell.cc

```diff
@@ -807,6 +807,212 @@ ReferenceCell::gmsh_element_type() const
 
 
 
+namespace
+{
+  double
+  distance_to_line_square(const Point<1> &x0,
+                          const Point<1> &x1,
+                          const Point<1> &p)
+  {
+    return std::min(x0.distance_square(p), x1.distance_square(p));
```

Comment:
That's not right. If the point is between x0 and x1, the distance should be zero.

Reply:
Fixed!

Reply:
It's not fixed. Did you forget to push?